### ISSUE TITLE
Feature Request: Automatically generate github Actions yaml to run tests in CI

### ISSUE THREAD

**Issue by thegostep** [nomiclabs/buidler#677]

**Comment by alcuadrado** [comment]
This is a really good idea. Maybe we should add an option for this to the sample project creation workflow.

We have to be sure that this won't cause problems to less experienced developers first though.

**Comment by thegostep** [comment]
Makes sense - I saw tsdx setup was doing this so thought it could work well here too
